src/components/BugzillaComponents/index.jsx

```diff
@@ -53,6 +53,7 @@ const BugzillaComponents = ({
                       onClick={onComponentDetails}
                       product={product}
                       component={component}
+                      teamkey={teamKey}
```

Comment:
It is not recommended to set custom attributes on core DOM nodes as these are a code smell for not letting React manage its own state.

Reply:
Let me know if the recent changes address your concerns or if I misunderstood it.

Reply:
So right now these exist as attributes on the `div` rendered into the DOM, and aren't passed as props to a component or function. Can these values be passed to `onComponentDetails` or similar?

```js
onClick={(e) => onComponentDetails(e, teamKey)}
```

Reply:
Does this commit address the two remaining concerns?
https://github.com/mozilla/bugzilla-dashboard/pull/27/commits/703f64a75f0684b8452a86755fdd30b0b1b382f8